.github/workflows/production-deploy.yml

```diff
@@ -19,7 +23,9 @@ jobs:
           script_stop: true
           script: |
             cd /home/akatsuki/pep.py
-            git pull origin main
+            git fetch origin
+            git checkout origin/${{ steps.extract_branch.outputs.branch }}
+            git pull origin ${{ steps.extract_branch.outputs.branch }}
```

Comment:
cc @tsunyoku to be replicated on the other repos as well - made a ticket https://osuakatsuki.atlassian.net/browse/AKAT-456